hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
The solution that I am testing and re-testing, keeps all the years. You will see situations where the updated copyright line looks like this ...
_* @copyright Copyright (C) 2012, 2013, 2017 DigitalGlobe (http://www.digitalglobe.com/)_